### PR TITLE
eilr examples + logic to switch examples based on loaded content

### PR DESCRIPTION
**Pull request by VivianAllen**
### What

- Add new examples for eilr release (see [trello ticket](https://trello.com/c/oazvZpZU))
- Add logic to switch examples based on what content is loaded (so don't load EILR examples unless you've loaded the EILR content!)
- make the arm / eilr examples react to the currently selected geography (default to examples in trello ticket if non selected)

### How to review

go to the preview below, checkout the EILR examples

### Who can review

Anyone

### CODE REVIEW

File: src/components/ArmEilrExamples.svelte [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+  import { page } from "$app/stores";
+  import { buildHyperlink } from "../helpers/buildHyperlinkHelper";
+  import { selected } from "../stores/selected";
+  export let reverseType = false;
+
+  $: ewNotSelected = $selected && $selected.geoType != "ew";
```
Comment (petem-methods) [inline src/components/ArmEilrExamples.svelte:7]: In JS you always use strict equality, `!==` and `===`

Comment (petem-methods) [reply src/components/ArmEilrExamples.svelte:7]: Also can use Optional Chaining operator for expressions likes this, like `$selected?.geoType !== "ew"`

**Review by petem-methods — CHANGES_REQUESTED** [review]
Great, one expression with comments!

File: src/components/ArmEilrExamples.svelte (hunk 1 above)
Comment (VivianAllen) [reply src/components/ArmEilrExamples.svelte]: fixed the operator

**Review by petem-methods — DISMISSED** [review]

**Pull request merged**